run/idp-sql/test/e2e_test_setup.yaml

```diff
@@ -0,0 +1,87 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+steps:
+
+- id: 'Create a dedicated database'
+  name: 'gcr.io/cloud-builders/gcloud'
+  entrypoint: '/bin/bash'
+  args:
+    - '-c'
+    - |
+      ./test/retry.sh "gcloud sql databases create ${_DB_NAME} \
```

Comment:
So ideally, we really should be reusing infrastructure as opposed to recreating / tearing down each time (see [External Resources](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#external-resources)).

Reply:
Realized now this is this database (and not the instance) but it looks like in the [Cloud SQL samples we reuse the database and just create/drop tables](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/d384d10314f89dba48b68f2874da7e1861a5f9f7/cloud-sql/mysql/sqlalchemy/connection_test.py#L81)

Reply:
See comment on current state of PR https://github.com/GoogleCloudPlatform/python-docs-samples/pull/5315#issuecomment-783866308

Using this method may make the tests pass, but it will also mean there's potential namespace collision, etc, because in this example the database is created in the client code. 

The nodejs version of this doesn't create/delete databases either https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/master/run/idp-sql/test/e2e_test_cleanup.yaml